packages/@aws-cdk/cloudformation-include/test/valid-templates.test.ts

```diff
@@ -251,6 +251,41 @@ describe('CDK Include', () => {
     );
   });
 
+  test('correctly change references to Conditions when renaming them', () => {
```

Comment:
I like the handling of two cases with one test here, nicely done.

Reply:
Thanks 🙂